docs/features/tuples_lists_records.md

```diff
@@ -0,0 +1,28 @@
+# Tuples, Lists, and Records
+
+```js
+// This will declare a simple tuple with a int and a char
+let tuplevalue:(int, char) = (1, \{a})
+
+// This will declare a record with a int called value and then get that from it
+let recordvalue:{ value: int } = {
+	value: 1 // There is either a newline or a ; at the end of this to separate these
```

Comment:
Perhaps "; to separate key-value pairs" to suggest its proper usage rather than to end a line